mysql-test/suite/galera/t/galera_wsrep_ddl_access_checking2.test

```diff
@@ -1470,6 +1470,67 @@ REVOKE CREATE ROUTINE,EXECUTE,ALTER ROUTINE ON testdb.* FROM 'testme'@'%';
 SHOW GRANTS FOR 'testme';
 FLUSH PRIVILEGES;
 
+--echo #
+--echo # SQLCOM_CREATE_SPFUNCTION (with log_bin_trust_function_creators=OFF) access test
+--echo #
+#
+# All tests are executed with log_bin_trust_function_creators=ON
+# (set in default_mysqld.cnf)
+# however OFF is the default value. Test CREATE FUNCTION with default
+# log_bin_trust_function_creators=OFF
+#
+--inc $test_id
+
+--echo # connection node_1
+--connection node_1
+set @old_log_bin_trust_function_creators= @@global.log_bin_trust_function_creators;
+set global log_bin_trust_function_creators=OFF;
+
+GRANT ALL PRIVILEGES ON testdb.* TO 'testme'@'%';
+--replace_regex /\*[0-9A-Z]+/<SECRET>/
+SHOW GRANTS FOR 'testme';
+FLUSH PRIVILEGES;
+
+--echo # connection node_testme
+--disconnect node_testme
+--connect node_testme, 127.0.0.1, testme, secret,, $NODE_MYPORT_1
+--connection node_testme
+use testdb;
```

Comment:
No need to use  `use testdb;` command as connection can directly be made to `testdb` by passing the db name to `--connect`
```suggestion
--connect node_testme, 127.0.0.1, testme, secret,testdb, $NODE_MYPORT_1
--connection node_testme
```

Reply:
This seems to be a convention of this file, so let's follow it.